Runtimes/Core/CMakeLists.txt

```diff
@@ -100,6 +101,7 @@ defaulted_option(SwiftCore_ENABLE_STDLIB_TRACING "Enable tracing in the runtime.
 option(SwiftCore_ENABLE_UNICODE_DATA "Include unicode data in Swift runtimes" ON)
 option(SwiftCore_ENABLE_SHORT_MANGLING_LOOKUPS "Build with fast-path context descriptor lookups based on well-known short manglings." ON)
 option(SwiftCore_ENABLE_FILESYSTEM_SUPPORT "Build for systems that have a filesystem" ON)
+option(SwiftCore_ENABLE_BACKTRACE_REPORTING "Build with backtrace reporting support" ${SwiftCore_HAS_BACKTRACE})
```

Comment:
This should also be enabled on Windows (which doesn't have `backtrace()`) and on ELF platforms where we use the unwinder instead of `backtrace()`… is that going to be handled in cache files rather than by logic here?